test/integration/apiserver/apiserver_test.go

```diff
@@ -1957,21 +1963,27 @@ func TestTransform(t *testing.T) {
 				t.Fatal(err)
 			}
 
-			rv, _ := strconv.Atoi(obj.GetResourceVersion())
-			if rv < 1 {
-				rv = 1
+			var rv string
+			if obj.GetResourceVersion() == "" {
+				rv = "0"
+			} else {
+				rv = previousRV
 			}
 
+			ctx, cancel := context.WithTimeout(context.Background(), wait.ForeverTestTimeout)
+			t.Cleanup(func() {
+				cancel()
```

Comment:
Is w.Close not closing the request?

Reply:
nope, I was curious, I'm in a crusade to reduce number of goroutine leakages on the apiserver/integration framework https://github.com/kubernetes/kubernetes/issues/108483  and I was curious if you found something wrong there

Reply:
> It is, but linters often complain about the `cancelFunc` being ignored. Happy to remove if necessary.

what linter?

Reply:
`gometalinter` for sure, whichever is built-in to GoLand/JetBrains as well. I think technically this case is fine since implicitly it will be cancelled when the close runs, but I figured it was good practice to be explicit.